src/providers/user/user.ts

```diff
@@ -10,19 +12,19 @@ import { Injectable } from '@angular/core';
 @Injectable()
 export class UserProvider {
 
-  regUrl: string = "https://j-reboot-backend.herokuapp.com/api/appUsers"
+  requestUrl: string = ENV.url
 
   constructor(public http: HttpClient) {
     console.log('Hello UserProvider Provider');
   }
 
   sendReg(user) {
     console.log('sendReg() runs')
-    return this.http.post(this.regUrl, user)
+    return this.http.post(this.requestUrl, user)
```

Comment:
This request URL needs to be modified to register. The `ENV.url` should just be some variation of: `"https://reboot-backend.herokuapp.com/api/` and + `appUsers/` to register. Test they both work on your local version.